diff --git a/filament/backend/include/backend/platforms/PlatformMetal.h b/filament/backend/include/backend/platforms/PlatformMetal.h
index 2d65454c3..33492f2e5 100644
--- a/filament/backend/include/backend/platforms/PlatformMetal.h
+++ b/filament/backend/include/backend/platforms/PlatformMetal.h
@@ -69 +69 @@ public:
-    virtual void createDevice(MetalDevice& outDevice) noexcept;
+    void createDevice(MetalDevice& outDevice) noexcept;
@@ -78 +78 @@ public:
-    virtual void createCommandQueue(
+    void createCommandQueue(
